Formula/pipenv.rb

```diff
@@ -0,0 +1,123 @@
+    sha256 "2c6a5de3089009e3da7c5dde64a141dbc8551d5b7f6cf4ed7c2568d0cc520a8f"
+  end
+
+  resource "mccabe" do
+    url "https://files.pythonhosted.org/packages/06/18/fa675aa501e11d6d6ca0ae73a101b2f3571a565e0f7d38e062eec18a91ee/mccabe-0.6.1.tar.gz"
+    sha256 "dd8d182285a0fe56bace7f45b5e7d1a6ebcbf524e8f3bd87eb0f125271b8831f"
+  end
+
+  resource "pathlib" do
+    url "https://files.pythonhosted.org/packages/ac/aa/9b065a76b9af472437a0059f77e8f962fe350438b927cb80184c32f075eb/pathlib-1.0.1.tar.gz"
+    sha256 "6940718dfc3eff4258203ad5021090933e5c04707d5ca8cc9e73c94a7894ea9f"
+  end
+
+  resource "pew" do
+    url "https://files.pythonhosted.org/packages/fb/f1/118e0293d33e824802cdf46662f4663ab4c561060ad484b061d4a2df8dbd/pew-0.1.27.tar.gz"
+    sha256 "8d41bc5ce058dcc28217ce958afef28b6e5998e4a599e554804afbe77b3a709e"
+  end
+
+  resource "pycodestyle" do
+    url "https://files.pythonhosted.org/packages/e1/88/0e2cbf412bd849ea6f1af1f97882add46a374f4ba1d2aea39353609150ad/pycodestyle-2.3.1.tar.gz"
+    sha256 "682256a5b318149ca0d2a9185d365d8864a768a28db66a84a2ea946bcc426766"
+  end
+
+  resource "pyflakes" do
+    url "https://files.pythonhosted.org/packages/5b/b7/dcd6ebc826065ca4ccd2406aac4378e1df6eb91124625d45d520219932a1/pyflakes-1.5.0.tar.gz"
+    sha256 "aa0d4dff45c0cc2214ba158d29280f8fa1129f3e87858ef825930845146337f4"
+  end
+
+  resource "requests" do
+    url "https://files.pythonhosted.org/packages/b0/e1/eab4fc3752e3d240468a8c0b284607899d2fbfb236a56b7377a329aa8d09/requests-2.18.4.tar.gz"
+    sha256 "9c443e7324ba5b85070c4a818ade28bfabedf16ea10206da1132edaa6dda237e"
+  end
+
+  resource "shutilwhich" do
+    url "https://files.pythonhosted.org/packages/66/be/783f181594bb8bcfde174d6cd1e41956b986d0d8d337d535eb2555b92f8d/shutilwhich-1.1.0.tar.gz"
+    sha256 "db1f39c6461e42f630fa617bb8c79090f7711c9ca493e615e43d0610ecb64dc6"
+  end
+
+  resource "urllib3" do
+    url "https://files.pythonhosted.org/packages/ee/11/7c59620aceedcc1ef65e156cc5ce5a24ef87be4107c2b74458464e437a5d/urllib3-1.22.tar.gz"
+    sha256 "cc44da8e1145637334317feebd728bd869a35285b93cbb4cca2577da7e62db4f"
+  end
+
+  resource "virtualenv" do
+    url "https://files.pythonhosted.org/packages/d4/0c/9840c08189e030873387a73b90ada981885010dd9aea134d6de30cd24cb8/virtualenv-15.1.0.tar.gz"
+    sha256 "02f8102c2436bb03b3ee6dede1919d1dac8a427541652e5ec95171ec8adbc93a"
+  end
+
+  resource "virtualenv-clone" do
+    url "https://files.pythonhosted.org/packages/36/66/d95f83255138ded1aec242de5ee51536226fa7b2582a7ce0863fea5dc4f2/virtualenv-clone-0.2.6.tar.gz"
+    sha256 "6b3be5cab59e455f08c9eda573d23006b7d6fb41fae974ddaa2b275c93cc4405"
+  end
+
+  def install
+    # `pipenv` needs to be able to find `virtualenv` and `pew` on $PATH. Rather
+    # than installing symlinks for those scripts into /usr/local/bin we leave
+    # them in `#{libexec}/bin` and make /usr/local/bin/pipenv a wrapper script
+    # which adds `#{libexec}/bin` to $PATH.
+    venv = virtualenv_create(libexec, "python")
```

Comment:
Let's not use virtualenv here. See `aws-shell` and `ipython@5` for examples.

Reply:
Will do.

I used a virtualenv on the advice of https://docs.brew.sh/Python-for-Formula-Authors.html. Is that document outdated? Is there a better reference anywhere?

Reply:
>Is that document outdated?

Yeah, we're steering away from virtualenv lately.

>Is there a better reference anywhere?

The older versions of the same document in the git commit history before the virtualenv stuff got added.